webdriverio/web/examples/typescript/test/pageobjects/login.page.ts

```diff
@@ -0,0 +1,23 @@
+import Page from './page';
+
+/**
+ * sub page containing specific selectors and methods for a specific page
+ */
+export class LoginPage extends Page {
+    /**
+     * a method to encapsule automation code to interact with the page
+     * e.g. to login using username and password
+     */
+    login (username: string, password: string) {
```

Comment:
In TS you should also provide return types to methods, so this should be

```ts
 login (username: string, password: string):void
```

You _need_ to do that for all page object methods